test/headless/context.js

```diff
@@ -54,4 +59,59 @@ export default class Context {
     this._execution = this._execution.wait(time);
     return this;
   }
+
+  onDOMReady() {
+    this.wait('.tracked-item[data-id="1"]');
+    return this;
+  }
+
+  waitForImpression(identifierNum) {
```

Comment:
Can we add a default timeout for these `waitFor` functions? Like if it took an entire second to complete, that might signal something was wrong.